chips/README.md

```diff
@@ -0,0 +1,294 @@
+    )
+}
+```
+
+Parameters | Descriptions
+-- | --
+`label: String` | Text to be displayed into the Chip
+`selected: Boolean,` | True if the Chip is selected, otherwise false
+`modifier: Modifier = Modifier` | `Modifier` to be applied to the component
+`enabled: Boolean = true` | True if you can click on the Chip, otherwise false
+`colors: ChipColors = VitaminChipColors.default()` | Colors to be applied to the Chip
+`sizes: ChipSizes = VitaminChipSizes.medium()` | Sizes to be applied to the Chip. (VitaminChipSizes.medium() | VitaminChipSizes.small())
+`ripple: RippleTheme` | The ripple effect to be applied to the Chip
+`onClick: (() -> Unit)?` | Callback to be called when the Chip is clicked
+`leadingIcon: (@Composable () -> Unit)` | Icon Content to be displayed before label into the Chip
+
+_In your project :_
+```kotlin
+VitaminChips.InputWithIcon(
+    label = "Walking",
+    onClick = {
+        // Callback to handle Chip state 
+    },
+    leadingIcon = {
+        Icon(
+            painter = painterResource(
+                id = R.drawable.ic_vtmn_walk_line
+            ),
+            contentDescription = null
+        )
+    }
+)
+```
+
+#### With Image
+
+```kotlin
+object VitaminChips {
+
+    @Composable
+    fun InputWithImage(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: ChipColors = VitaminChipColors.default(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminChipRipples.default(),
+        onClick: () -> Unit,
+        leadingContent: (@Composable () -> Unit)
+    )
+}
+```
+
+Parameters | Descriptions
+-- | --
+`label: String` | Text to be displayed into the Chip
+`selected: Boolean,` | True if the Chip is selected, otherwise false
+`modifier: Modifier = Modifier` | `Modifier` to be applied to the component
+`enabled: Boolean = true` | True if you can click on the Chip, otherwise false
+`colors: ChipColors = VitaminChipColors.default()` | Colors to be applied to the Chip
+`sizes: ChipSizes = VitaminChipSizes.medium()` | Sizes to be applied to the Chip. (VitaminChipSizes.medium() | VitaminChipSizes.small())
+`ripple: RippleTheme` | The ripple effect to be applied to the Chip
+`onClick: (() -> Unit)?` | Callback to be called when the Chip is clicked
+`leadingContent: (@Composable () -> Unit)` | Content to be displayed before label into the Chip
+
+_In your project :_
+```kotlin
+VitaminChips.InputWithImage(
+    label = "Walking",
+    onClick = {
+        // Callback to handle Chip state 
+    },
+    leadingContent = {
+        Image(
+            painter = painterResource(R.drawable.vtmn_flag_fr),
+            contentScale = ContentScale.Crop,
+            alignment = Alignment.Center,
+            contentDescription = null
+        )
+    }
+)
+```
+
+### Single Choice Chip
+
+Single choice chips allow the selection of a single chip from a set of options.
+
+```kotlin
+@Composable
```

Comment:
you forgot to show the Vitamin object class.

Reply:
Added